packages/login-with-reapit/src/components/logout.svelte

```diff
@@ -23,5 +25,3 @@
     background-color: #23a4de;
   }
 </style>
-
-<button on:click={handleLogoutClick}><SignOutButton /></button>
```

Comment:
why the rearrange here? (just curious)

Reply:
Will's change, makes sense as you want the element below the style tags